cheatsheets/Content_Security_Policy_Cheat_Sheet.md

```diff
@@ -139,18 +158,20 @@ If the [upgrade-insecure-requests](https://developer.mozilla.org/en-US/docs/Web/
 
 - To prevent all framing of your content use:
   - `Content-Security-Policy: frame-ancestors 'none';`
-- To allow for your site only, use:
+- To allow for the site itself, use:
   - `Content-Security-Policy: frame-ancestors 'self';`
-- To allow for trusted domain , do the following:
+- To allow for trusted domain, do the following:
   - `Content-Security-Policy: frame-ancestors trusted.com;`
 
-## Loading Files from CDN
+## Strict Policy
```

Comment:
You highlight that the aim of this policy is to mitigate XSS. And to get best coverage of sinks they can use:
 - strict CSP for classical stored/reflected XSS and parts of DOM XSS
 - trusted types for DOM XSS

Reply:
Trusted types are still in the testing phase. Do you think that it is safe to include them in this cheat sheet already?

Reply:
According to this [post](https://developers.google.com/web/updates/2019/02/trusted-types#get-started): 

*We are working on providing more code examples, guides and documentation on how to migrate applications to Trusted Types. We feel this is the right moment for the web developer community to start experimenting with it.*

As CS must be directly usable for dev team for production level application, I think that we should keep it aside (and open an issue on it to remember to add it) until a they are declared ready for production.

By the way, I have tested the [demo page](https://wicg.github.io/trusted-types/demo/):
* All tests are green on the last version of Firefox and Chrome and Opera on a WIN10 Pro.
* Tests n°4 and n°12 are red on Edge on a WIN10 Pro.
* Page do not load in IE11 with error `SCRIPT5009: 'Reflect' is undefined` on a WIN10 Pro.

I am not blocked on this position, it's just my opinion 😃

Reply:
sgtm! adding a todo or reminder sounds like good idea.

Reply:
I will create a issue for that point if @ThunderSon is agree.

Reply:
sgtm: Sounds good to me (Lukas). Sure thing, go ahead!

Reply:
Thank you for the explanation 😃 
I have created the issue #95 about `Trusted Types`.